src/solver/conv_winoRxS.cpp

```diff
@@ -413,115 +413,213 @@ ConvBinWinoRxS<Winodata, Winofilter>::Search(const ConvolutionContext& ctx,
 template <int Winodata, int Winofilter>
 class ShaderModel : public UnifiedDescriptionConv2d
 {
-    const size_t DATATYPE_BITS;    // S
-    const size_t n_groups;         // BQ ~compute units
-    const bool out_of_model_scope; // Shader model produces unreliable results.
+    static_assert(IS2X3 || IS3X2, "Unsupported filter configuration");
```

Comment:
Sure, only IS2X3 or IS3X2 are supported.
IS2X3 and IS3X2 are mutual exclusive and only IS2X3 can be true or only IS3X2 can be true:
`#define IS2X3 (Winodata == 2 && Winofilter == 3)`
`#define IS3X2 (Winodata == 3 && Winofilter == 2)`

Reply:
I'll get rid of this static assert here and move it inside convolution.
Moreover we've decided to remove template parameters, so it can't be static anymore.